4.5 Error event docs (#7554)
cakephp/docs: en/development/errors.rst

        }
    );

Within an ``Error.beforeRender`` handler you have a few options:

> * Stopping the event will prevent rendering.

Reviewer: ```suggestion
* Stop the event to prevent rendering.
```

> * Returning a string value will skip rendering and render the content generated
>   by the listener.

Reviewer: ```suggestion
* Return a string to skip rendering and use the provided string instead
```

Within an ``Exception.beforeRender`` handler you have a few options:

> * Stopping the event will prevent rendering.

Reviewer: ```suggestion
* Stop the event to prevent rendering.
```

> * Setting the ``exception`` data attribute with ``setData('exception', $err)``
>   will replace the exception that is being rendered.

Reviewer: ```suggestion
* Set the ``exception`` data attribute with ``setData('exception', $err)``
  to replace the exception that is being rendered.
```

> * Returning a response from the event listener will skip rendering and use the
>   provided response as the error page.

Reviewer: ```suggestion
* Return a response from the event listener to skip rendering and use
  the provided response instead.
```
